infrastructure/src/main/resources/logback.prod.xml

```diff
@@ -20,7 +23,13 @@
         </encoder>
     </appender>
 
+    <!-- https://logback.qos.ch/manual/appenders.html#AsyncAppender -->
```

Comment:
Why don't you also use `ImmediateFlush=false` and `BufferedIO`

Reply:
Since we have AsyncAppender what's the point for immediateFlush=false?
Didn't see BufferedIO in the documentation, can't say anything